devolving_music/models/song_submission.py

```diff
@@ -31,7 +50,61 @@ def voteable(self) -> bool:
     def get_voteable_submissions(Event):
         voteable_submissions = [
                 sub
-                for sub in SongSubmission.objects.filter(event__exact=Event).order_by('?')
+                for sub in SongSubmission.objects.filter(event__exact=Event).order_by('id')
                 if sub.voteable()
             ]
         return voteable_submissions
+
+    @staticmethod
+    def check_compare(comparison_submission:"SongComparison",\
+        song1:"SongSubmission",song2:"SongSubmission"):
+        return not song1.compare_present(comparison_submission) and \
+        not song2.compare_present(comparison_submission)
+
+    @staticmethod
+    def elo_song_rating(comparison_submission:"SongComparison",song1,song2,score_range=30):
+        if SongSubmission.check_compare(comparison_submission,song1,song2):
+            song1.update_info(comparison_submission)
+            song2.update_info(comparison_submission)
+            song1.quality_score,song2.quality_score=\
+                elo_rating(song1.quality_score,song2.quality_score,\
+                score_range,comparison_submission.first_better)
+            song1.energy_score,song2.energy_score=\
+                elo_rating(song1.energy_score,song2.energy_score,\
+                score_range,comparison_submission.first_peakier)
+            song1.post_peak_score,song2.post_peak_score=\
+                elo_rating(song1.post_peak_score,song2.post_peak_score,\
+                score_range,comparison_submission.first_post_peakier)
+            return song1,song2
+        else:
+            return song1,song2
+
+    @staticmethod
+    def submission_index(song_submissions:Iterable["SongSubmission"], song:"SongSubmission"):
```

Comment:
I'd prefer not to name variables with `SongSubmission` type `song`, it might be confusing since there's a separate `song` model.